docker/web-and-data/Dockerfile

```diff
@@ -42,10 +42,11 @@ FROM openjdk:8-jre
 # download system dependencies first to take advantage of docker caching
 RUN apt-get update; apt-get install -y --no-install-recommends \
         build-essential \
-        libmysql-java \
+        default-mysql-client \
```

Comment:
Thanks so much for sending this PR! I needed this one line, saved me some work debugging why the docker build was failing 🙏